apps/console/src/features/groups/components/wizard/group-assign-users.tsx

```diff
@@ -199,15 +200,15 @@ export const AddGroupUsers: FunctionComponent<AddGroupUserProps> = (props: AddGr
 
     useEffect(() => {
         if (userListMetaContent) {
-            const attributes = generateAttributesString(userListMetaContent.values());
+            const attributes: string = generateAttributesString(userListMetaContent.values());
 
             getList(listItemLimit, 0, null, attributes, userStore);
 
         }
-    }, [ listItemLimit ]);
+    }, [ listItemLimit, userStore ]);
 
-    const handleUnassignedItemCheckboxChange = (role) => {
-        const checkedGroups = [ ...checkedUnassignedListItems ];
+    const handleUnassignedItemCheckboxChange = (role: any) => {
```

Comment:
Let's use proper type here, instead of `any`. Please check the other places as well.